mmdet/models/roi_heads/bbox_heads/multi_instance_bbox_head.py

```diff
@@ -0,0 +1,615 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+from mmengine.config import ConfigDict
+from mmengine.structures import InstanceData
+from torch import Tensor, nn
+
+from mmdet.models.roi_heads.bbox_heads.bbox_head import BBoxHead
+from mmdet.models.task_modules.samplers import SamplingResult
+from mmdet.models.utils import empty_instances
+from mmdet.registry import MODELS
+from mmdet.structures.bbox import bbox_overlaps
+
+
+@MODELS.register_module()
+class MultiInstanceBBoxHead(BBoxHead):
+    r"""Bbox head used in CrowdDet.
+
+    .. code-block:: none
+                                        /-> cls convs_1 -> cls fcs_1 -> cls_1
+                                   |----
+                                   |    \-> reg convs_1 -> reg fcs_1 -> reg_1
+                                   |
+                                   |    /-> cls convs_2 -> cls fcs_2 -> cls_2
+        shared convs -> shared fcs |----
+                                   |    \-> reg convs_2 -> reg fcs_2 -> reg_2
+                                   |
+                                   |                     ...
+                                   |
+                                   |    /-> cls convs_k -> cls fcs_k -> cls_k
+                                   |----
+                                        \-> reg convs_k -> reg fcs_k -> reg_k
+
+    """  # noqa: W605
+
+    def __init__(self,
+                 num_instance: int = 2,
+                 refine_flag: bool = True,
```

Comment:
The basic setting does not use refine module. And the word 'flag' can be removed. 
```suggestion
                 with_refine: bool = False,
```